docs/schema/v0/flag_manifest.json

```diff
@@ -0,0 +1,122 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$defs": {
+        "flag": {
+            "oneOf": [
+                {
+                    "$ref": "#/$defs/boolType"
+                },
+                {
+                    "$ref": "#/$defs/stringType"
+                },
+                {
+                    "$ref": "#/$defs/intType"
```

Comment:
Do we distinguish between int and float elsewhere in OF? I've only seen things like `getNumberValue`, but that might be because I'm mostly working in typescript...

I suppose in the spec [we do say](https://openfeature.dev/specification/types#number) that implementation languages can get more specific, so this would be a way of allowing that. We could also achieve that by supporting the ability to [define a flag values schema](https://docs.google.com/document/d/19GQpSNcLnDEbzJRL6FpzKHFEQWVGNrgNK25lC4-JuE8/edit?disco=AAABP8cdSoo) though, rather than supporting two distinct types here.

Reply:
Yes, we do have a distinction between ints and floats in some languages. I think we'll need it here, but we'll need to keep in mind that we'll need to combine them when generating a TypeScript client.

Reply:
I'm also not sure what is the best path here - for me, I added the separation based on the fact that https://pkg.go.dev/github.com/open-feature/go-sdk@v1.12.0/openfeature#pkg-examples golang accessors also explicitly lists the separation between int and floating type - but for example, `flagd` treats both the same (https://flagd.dev/schema/v0/flags.json).
For a codegen, I believe we should be have access to this granularity.

Reply:
JSON doesn't have a distinction between ints and floats either, but I agree we need this in a codegen tool. It should be straightforward to handle codegen in languages that don't care about this distinction.